File: app/src/main/java/temple/edu/yelp_randomizer/View/ReviewsView.java
package temple.edu.yelp_randomizer.View;

import android.view.View;
import android.widget.ImageView;
import android.widget.TextView;
import androidx.annotation.NonNull;
import androidx.recyclerview.widget.RecyclerView;
import temple.edu.yelp_randomizer.R;

public class ReviewsView extends RecyclerView.ViewHolder {

    private TextView reviewNameTextView;
    private TextView reviewRatingTextView;
    private TextView reviewTextTextView;
    private ImageView profilePictureImageView;

    public ReviewsView(View itemView) {
        super(itemView);

        reviewNameTextView = itemView.findViewById(R.id._reviewNameTextView);
        reviewRatingTextView = itemView.findViewById(R.id._reviewRatingTextView);
        reviewTextTextView = itemView.findViewById(R.id._reviewTextTextView);
        profilePictureImageView = itemView.findViewById(R.id._reviewProfilePictureImageView);
    }

    public TextView getReviewNameTextView() {
        return reviewNameTextView;
    }

    public TextView getReviewRatingTextView() {
        return reviewRatingTextView;
    }

    public TextView getReviewTextTextView() {
        return reviewTextTextView;
    }

    public ImageView getProfilePictureImageView() {
        return profilePictureImageView;
    }


}

File: app/src/main/java/temple/edu/random/View/ReviewsView.kt
package temple.edu.random.View

import android.widget.TextView
import android.view.View
import android.widget.ImageView
import temple.edu.random.R
import androidx.recyclerview.widget.RecyclerView

class ReviewsView(itemView: View) : RecyclerView.ViewHolder(itemView) {
    val reviewNameTextView: TextView
    val reviewRatingTextView: TextView
    val reviewTextTextView: TextView
    val profilePictureImageView: ImageView

    init {
        reviewNameTextView = itemView.findViewById(R.id._reviewNameTextView)
        reviewRatingTextView = itemView.findViewById(R.id._reviewRatingTextView)
        reviewTextTextView = itemView.findViewById(R.id._reviewTextTextView)
        profilePictureImageView = itemView.findViewById(R.id._reviewProfilePictureImageView)
    }
}